app/src/backend/dependency.ts

```diff
@@ -162,6 +170,13 @@ export function installAnaconda(
   let installAnaconda = () => {
     logging.debug('  [*] Installing anaconda.');
     progressCb(30, 'Installing...');
+    let durationInitSJCloud = process.hrtime(startInitSJCloud);
+    logging.silly(
+      `=== Initializing SJ Cloud took ${
+        durationInitSJCloud[0]
+      }s and ${durationInitSJCloud[1] / 100000}ms to run ===`
```

Comment:
If you want to keep consistency, log lines seem to be mostly denoted with "  [*]", and section titles are wrapped with "==".

Logging "x s and y ms" is also weird. Combine them for either seconds or milliseconds.

Reply:
Sure, I can change the logging output. I wanted to be able to see the times distinctly from the other logs (which were logging steps/states) but its better to just be consistent.

I actually chose `process.hrtime` because "x s and y ms" was more human readable while still being precise. I can see why seconds only would be enough (since we're not really trying to shave milliseconds off the times), but would you ever want to have milliseconds only? I'm just wondering..

Reply:
I meant combine them for the output, e.g., if you had [8, 210], you'd display it as "8.21 s" or "8210 ms".